docs/troubleshooting/FAQ.mdx

```diff
@@ -384,11 +491,11 @@ For developers updating ESLint rules in plugins that still need to support types
 
 For more context, see the [Some properties named typeParameters instead of typeArguments](https://github.com/typescript-eslint/typescript-eslint/issues/146) issue, and the implementing [fix: rename typeParameters to typeArguments where needed](https://github.com/typescript-eslint/typescript-eslint/pull/5384) pull request.
 
-The [typescript-eslint v6 release post](https://deploy-preview-6515--typescript-eslint.netlify.app/blog/announcing-typescript-eslint-v6-beta) has more information on typescript-eslint v6.
+The [typescript-eslint v6 release post](/blog/announcing-typescript-eslint-v6-beta) has more information on typescript-eslint v6.
```

Comment:
[Non-Actionable] Heh, yeah, I think we're ready to refer to it as a stable production link 😂